src/views/PropTableView.vue

```diff
@@ -250,7 +256,16 @@ function getChildren() {
                     child.link = q.object.value;
                 } else if (q.predicate.value === qname("a")) {
                     child.type = q.object.value;
-                }
+                } else if (item.value.type === qname("dcat:Catalog") && q.predicate.value === qname("dcterms:publisher")) {
+                    const publisher: ListItemSortable = { label: q.object.value };
+                    child.extras.publisher = publisher;
+                } else if (item.value.type === qname("dcat:Catalog") && q.predicate.value === qname("dcterms:creator")) {
+                    const creator: ListItemSortable = { label: q.object.value };
+                    child.extras.creator = creator;
+                } else if (item.value.type === qname("dcat:Catalog") && q.predicate.value === qname("dcterms:issued")) {
+                    const publisher: ListItemSortable = { label: q.object.value };
+                    child.extras.publisher = publisher;
```

Comment:
This should be assigned to `child.extras.issued` instead of `publisher`.

Reply:
Ah whoops, good catch. Fixed in latest commit